scripts/automation/release/packaged.py

```diff
@@ -0,0 +1,112 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+#  pylint: disable=line-too-long
+
+from __future__ import print_function
+
+import argparse
+import os
+import sys
+import tempfile
+import tarfile
+import shutil
+from subprocess import check_call, check_output
+
+from .version_patcher import VersionPatcher
+from ..utilities.path import get_all_module_paths, get_repo_root
+
+REPO_ROOT_DIR = get_repo_root()
+COMPLETION_FILE = os.path.join(REPO_ROOT_DIR, 'packaged_releases', 'az.completion')
+ARCHIVE_FILE_TMPL = 'azure-cli_packaged_{}'
+
+class Patch(object):
+    def __init__(self, src_of_patch, path_to_patch):
+        """
+        - src: Relative path from the repo root
+        - dest: Relative path to file to patch in the packaged release
+        """
+        self.src_of_patch = src_of_patch
+        self.path_to_patch = path_to_patch
+
+    def apply(self, base_dir):
+        src = os.path.join(REPO_ROOT_DIR, self.src_of_patch)
+        dest = os.path.join(base_dir, self.path_to_patch)
+        shutil.copy(src, dest)
+
+PATCHES = [
+    Patch(os.path.join('packaged_releases', 'patches', 'patch_pkg_util.py'),
```

Comment:
By Design.
The paths here are supposed to be relative (see code above).
The first path is relative path from the repo root, the second is relative path to file to patch in the packaged release.
The REPO_ROOT_DIR is added to the first path, and the current working dir is added to the second path.